packages/spelunker-web/src/components/images/GameImage/index.jsx

```diff
@@ -33,7 +35,7 @@ const GameImageElement = (props) => {
 };
 
 const GameImage = (props) => {
-  const encoded = encodeURIComponent(props.file);
+  const encoded = encodeURI(toPipelinePath(props.file));
```

Comment:
I switched to `encodeURI`, since this is producing part of a path, and not part of a query parameter. `encodeURIComponent` will convert `/` to `%2F`, which isn't what we want.

Reply:
👍  This was originally used to ensure Firefox wouldn't break over backslashes.

Reply:
`encodeURI` still encodes backslashes, for what it's worth.

Reply:
Hah, never mind. I ran a test and thought that was the reasoning behind it. My test was wrong 😊